api/src/main/java/org/openmrs/api/impl/ObsServiceImpl.java

```diff
@@ -17,10 +17,7 @@
 import org.openmrs.Patient;
 import org.openmrs.Person;
 import org.openmrs.aop.RequiredDataAdvice;
-import org.openmrs.api.APIException;
-import org.openmrs.api.EncounterService;
-import org.openmrs.api.ObsService;
-import org.openmrs.api.PatientService;
+import org.openmrs.api.*;
```

Comment:
We discourage wild card imports as per https://wiki.openmrs.org/display/docs/Coding+Conventions